app/services/update_metadata_from_proquest_file.rb

```diff
@@ -33,7 +33,7 @@ def attributes
 
   def embargo_start_date
     if attributes[:DISS_agreement_decision_date].blank?
-      transformed_start_date
+      transformed_start_date(attributes)
```

Comment:
You shouldn't need to change these methods to pass the attributes hash around.  There is an attributes ivar that already contains them, so you should just be able to directly call the attributes method whenever you need them.  The attributes method is here:
https://github.com/curationexperts/alexandria-v2/blob/master/app/services/update_metadata_from_proquest_file.rb#L27-L32